### ISSUE TITLE
Question: how to change Language?

### ISSUE THREAD

**Issue by odo2063** [OpenBoard-org/OpenBoard#249]
Hello!

How do change the Language in Linux?(Ubuntu 18.04)

The '-lang de' does not have any effect. 
I changed the fallback in the source code but it is still in english.

Also changing language in AppImage would be nice.

kind regards
odo2063

**Comment by kaamui** [comment]
Hello,

I just tried with `openboard -lang de` and it worked for me. 

If you built OpenBoard yourself, please verify that you copied on your build directory the i18n folder (you can find it on the `resources` folder)

**Comment by odo2063** [comment]
Sadly this doesn't work :-/
```
linuxadmin@testrechner3:~/openboard/OpenBoard/build/linux/release/product$ ls -l
insgesamt 8940
drwxrwxr-x 2 linuxadmin linuxadmin    4096 Nov  3 12:13 etc
drwxrwxr-x 2 linuxadmin linuxadmin    4096 Nov  3 11:51 i18n
drwxrwxr-x 3 linuxadmin linuxadmin    4096 Nov  3 12:13 library
-rwxrwxr-x 1 linuxadmin linuxadmin 9138848 Nov  3 12:04 OpenBoard

```

**Comment by odo2063** [comment]
I do now have found the pre-release package for 18.04....evrything is fine with it...except for some translations...

**Comment by odo2063** [comment]
As for others may want to find it too....
[](https://github.com/OpenBoard-org/OpenBoard/releases/tag/v1.5.3-ubuntu18)

**Comment by kaamui** [comment]
It was initially announced here  : https://github.com/OpenBoard-org/OpenBoard/issues/231

Note that there is an issue with font changes ignored, no SWF support and desktop mode not working on gnome. 

The issue on fonts as been resolved though (fdecd32). If you're building your own version, you should cherry-pick this commit to add it to the master (1.5.3) or dev (1.5.4) branch, especially if you're working on Ubuntu 18.04.
